src/OpenTK/Platform/Windows/WinGraphicsMode.cs

```diff
@@ -252,7 +253,68 @@ private static AccelerationType GetAccelerationType(ref PixelFormatDescriptor pf
             }
             return type;
         }
+        GraphicsMode SelectGraphicsModePFD(IntPtr device, ColorFormat color, int depth, int stencil, int samples, ColorFormat accum,
```

Comment:
Use explicit access modifiers, and document the method. What is PFD? Don't use abbreviations.